src/main/java/com/rapid7/net/Utils.java

```diff
@@ -0,0 +1,20 @@
+package com.rapid7.net;
+
+/**
+ * Utility class to avoid including external dependencies.
+ * 
+ * @since 1.1.41
```

Comment:
```suggestion
 * @since 3.0.1
```

Think we're good to merge otherwise 😄